package.json

```
@@ -98,7 +98,6 @@
   },
   "jupyterlab": {
     "extension": true,
```

We would also need to add the following here to say @jupyter-widgets/base is shared and not bundled by yjs-widgets:
```suggestion
    "extension": true,
    "sharedPackages": {
      "@jupyter-widgets/base": {
        "bundled": false,
        "singleton": true
      }
    }
```